ratis-server/src/main/java/org/apache/ratis/server/raftlog/segmented/SegmentedRaftLogWorker.java

```diff
@@ -355,7 +356,7 @@ private boolean shouldFlush() {
     } else if (pendingFlushNum >= forceSyncNum) {
       return true;
     }
-    return pendingFlushNum > 0 && queue.isEmpty() && lastFlush.elapsedTime().compareTo(flushIntervalMin) > 0;
```

Comment:
> @SincereXIA , We already have raft.server.log.flush.interval.min . Do you mean adding another flush interval?

In my test, I found that add flush interval in this way has bug, It will greatly reduce the submission speed of Raft log. So I had to remove it. 

For now, I can't increase the refresh interval.